src/vector/vector2d.rs

```diff
@@ -41,11 +41,41 @@ impl<C> Vector2d<C>
 where
     C: Component,
 {
+    /// Initializes a new instance of the `Vector2d` struct.
+    pub fn new(x: C, y: C) -> Self {
+        Self { x, y }
+    }
```

Comment:
Its members are all public. It doesn't need a constructor.

If anything, it's clearer to have explicit `x: ..., y: ...`

Reply:
Got it! - removed it. I had originally added it for symmetry with other crates such as `nalgebra`. Personally I find it more concise to create typical vectors like this, but ultimately I don't mind.

Reply:
Maybe PR it separately